universal-login-example/src/components/PendingAuthorizations.js

```diff
@@ -10,7 +10,7 @@ import {utils} from 'ethers';
 
 const addKeyPaymentOptions = {
   gasToken: tokenContractAddress,
-  gasLimit: 1,
+  gasLimit: 1000000,
   gasPrice: utils.parseEther('0.00011').toString()
 };
```

Comment:
that should be stored in the config file. 
config.json:
```js
defaultGasOptions: {
  gasLimit: 1000000,
  gasPrice: utils.parseEther('0.00011').toString()
}
```

and when using:
```js
const addKeyPaymentOptions = {...defaultGasOptions, gasToken: tokenContractAddress};
await sdk.addKey(to, publicKey, privateKey, addKeyPaymentOptions);

```

Reply:
✅